### DISCUSSION TITLE
Does it support beta endpoints?

### DISCUSSION THREAD

**Discussion by divyeshjesadiya** [dcblogdev/laravel-microsoft-graph#57] [General]
I can make V1.0 call with it but not beta endpoints. Any how does it support beta endpoints?

**Comment by dcblogdev** [comment]
I've just added a method setApiVersion to set API version on call defaults to 1.0 

`MsGraph::setApiVersion('beta')->get('me')`

I've released this with `v3.2.2` this will allow you to set the API version directly for the duration of a single call.
